### ISSUE TITLE
Zitadel upgrades

### ISSUE THREAD

**Issue by srtalbot** [cds-snc/platform-forms-client#4288]
# Summary
Upgrade to latest Zitadel stable version.

**Comment by patheard** [comment]
After the `v2.62.0` upgrade, Zitadel was not able to read its configuration from the `config.yaml` file.  This is likely a regression, so we upgraded to `v2.61.1` and will wait for a fix.

**Comment by patheard** [comment]
Following the upgrade, we discovered several errors that were not directly related to the upgrade:

1. Service users were not being deleted from forms by the App.  This meant that if a new service user was created for a form that had previously had one, a duplicate user exception was thrown.  Addressed in (https://github.com/cds-snc/platform-forms-client/pull/3908) 
1. The App's access token expiry was set to [60 minutes by the `@zitadel/node` dep](https://github.com/smartive/zitadel-node/blob/c481ef608530b80deddfc47ac0a0d59733a8fcec/src/api/interceptors.ts#L40-L41), but Zitadel had this token expiry at 30 minutes, which was causing intermittent `token expired` errors.  This has been fixed by setting the token timeout in the Zitadel config.  @bryan-robitaille is looking at creating our own interceptor for `@zitadel/node` so we can set whatever expiry time we want.
1.  [Several errors around being `unable to set instance` for requests](https://gcdigital.slack.com/archives/C05G75MNP4Z/p1726661372764769) were tracked down to clients closing their connection before Zitadel had a chance to respond.  This type of `context canceled` error has now been suppressed to remove noise from the event channels.
